girder/api/docs.py

```diff
@@ -53,8 +54,13 @@ def _toOperation(info, resource, handler):
     """
     Augment route info, returning a Swagger-compatible operation description.
     """
+    module_name = inspect.getmodule(handler).__name__
+    if module_name.startswith('girder.plugins'):
+        module_tag = 'plugin:' + module_name.split('.')[2]
```

Comment:
@Xarthisius  Sorry for the delay in reviewing this PR. This check looks a little fragile, in fact it will be broken by Girder 3.0 as plugins are moving to being installed using pip. It seems like your use case is to tags plugins that you have added to a Girder instance. So instead of automatically tagging all plugins ( based on module name) could we instead add API to explicitly add tags to a plugins endpoints, for example add a parameter to the describe route decorator?